index.html

```diff
@@ -164,11 +169,10 @@ <h2>Profesional Art Printing Data</h2>
                             </div>
                         </div>
                     </div>
-                    <button class="btn btn-style" type="button">See Project</button>
                 </div>
             </div>
         </div>
```

Comment:
[REQUIRED] 
- [ ] Please make sure your cards are lined up symmetrically just like the way it is  in the [chosen Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). As a future web developer, you will have to build UI based on the design specified in the technical requirements, let's get used to it from the scratch :relaxed: 

The design implemented by you:
![Screenshot from 2022-02-17 17-40-59](https://user-images.githubusercontent.com/67608876/154484702-4f090051-58dc-471b-8db0-0e4f5895d13c.png)

[Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
![Screenshot from 2022-02-17 17-48-02](https://user-images.githubusercontent.com/67608876/154484908-070a5eaa-92af-480d-9b65-f44472712b7f.png)

- [ ] Please kindly make sure the `<button/>` - `See Project` is placed at the end of the `card` vertically.
- [ ] Also please kindly make sure your `mobile-version` is not getting affected by the changes your implement for the `desktop-version`, in this particular case, `mobile-version` of the web page is affected by the current changes. Please take a look at the screenshot below:

The current look of the mobile version from the `web-view` branch:
![Screenshot from 2022-02-17 17-53-49](https://user-images.githubusercontent.com/67608876/154486075-615bba49-b146-472d-bbd6-3fa6304d7d15.png)

The stable look of the mobile version from the `master` branch:
![Screenshot from 2022-02-17 17-59-03](https://user-images.githubusercontent.com/67608876/154486723-f9170659-5d47-4edc-b3ee-76584bda47de.png)

_hint: I can see that you are actually changing your [index.html](https://github.com/Regiss05/Portfolio-setup-and-mobile-version-skeleton/blob/web-view/index.html) to apply the `responsiveness`. However, please kindly note that `responsiveness` is best implemented in `CSS` file by using [`@media queries`](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) in a much easier and straightforward way_